src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -906,6 +683,10 @@ private UpdateSite.Data scanWAR(File war, @NonNull Map<String, String> pluginGro
                                 shortName = m.group(1);
                             }
                         }
+                        String longName = manifest.getMainAttributes().getValue("Long-Name");
+                        if (longName != null) {
+                            plugin.put("title", longName);
+                        }
```

Comment:
Unrelated bug fix while I was here: the non-WAR modes correctly parsed the title, but WAR mode never did. This doesn't affect production logic, but it is used for display purposes. Using a proper title makes it easier to read the logs.